apps/documentation/docs/luos-technology/index.mdx

```diff
@@ -7,24 +7,39 @@ custom_edit_url: null
 ## The most for the developer
 
 Luos provides a simple way to think your hardware products as a group of independant features. You can easily manage and share your hardware products' features with your team, external developers, or with the community.
-Luos engine is an open-source lightweight library that can be used on any MCU, leading to free and fast multi-electronic-boards products development. Choosing Luos to design a product will help you to develop, debug, validate, monitor, and manage it from the cloud.
+Luos engine is an <a href="https://github.com/Luos-io/luos_engine" target ="_blank" rel="external nofollow">open-source lightweight library</a> that can be used on any [MCU](../compatibility/mcu_demoboard), leading to free and fast multi-electronic-boards products development. Choosing Luos to design a product will help you to develop, debug, validate, monitor, and manage it from the cloud.
+
+<a href="/tutorials/get-started" class="pagination-nav__link" style={{ display: 'inline-block' }}>
```

Comment:
Did you forget the `target="_blank"` here ? Else why don't use a Markdown link ?